crt_portal/cts_forms/templates/forms/report_data.html

```diff
@@ -0,0 +1,163 @@
+<div class="crt-portal-card">
+    <div class="crt-portal-card__content margin-bottom-3">
+        {% include "forms/snippets/preview_button.html" with step_name="contact" step_value=0 %}
+
+        <h2 class="inline-block">{{ordered_step_names.0}}</h2>
+
+        <h3>{{ question.contact.contact_name_title }}</h3>
+
+        <h4>{{ question.contact.contact_first_name }}</h4>
+        <p>{{ form_data_dict.contact_first_name }}</p>
+        <h4>{{ form_data_dict.contact_last_name }}</h4>
```

Comment:
As-is this render's the respondents `contact_last_name` value twice, I think this line was intended to be `question.contact.contact_last_name`